### ISSUE TITLE
v3 not work

### ISSUE THREAD

**Issue by xpader** [amphp/mysql#116]
amphp/mysql:v3 not work.

```
Fatal error: Declaration of Amp\Mysql\Link::beginTransaction(int $isolation = Amp\Mysql\Transaction::ISOLATION_COMMITTED): Amp\Mysql\Transaction must be compatible with Amp\Sql\Link::beginTransaction(Amp\Sql\TransactionIsolation $isolation = Amp\Sql\TransactionIsolation::COMMITTED): Amp\Sql\Transaction in /home/pader/apps/wind-framework-new/example/vendor/amphp/mysql/src/Link.php on line 14
```

**Comment by trowski** [comment]
The v3 branch has been updated and should work again.

Please be aware this is under active development and is not tagged, so there may be unexpected BC breaks.

**Comment by xpader** [comment]
I can't install amphp/mysql:3.x-dev beacause it fixed revolt/event-loop to dev-main as 0.2.3..
